addons/website/static/src/scss/website.scss

```diff
@@ -505,6 +505,13 @@ font[class*='bg-'] {
     }
 }
 
+// Highlight '.btn-group's active buttons
+.btn-group .btn-light.active {
+    box-shadow: inset 0 0 0 $border-width o-color('primary');
+    border-color: transparent;
+    background-color: mix(o-color('primary'), $light, 10%);
+}
+
```

Comment:
> Should it be in bootstrap_review?
Yes 👍 

> In some circumstances, the default BS design for light active buttons is not visible enough. This combination achieves a visible result that is not too "invasive".

Reply:
Using $component-active-bg instead of $primary, that way it works when changing btn-primary color only (and not o-color-1). Might need to be o-cc dependant in the future :shrug: